lib/src/query_component.html

```diff
@@ -1,17 +1,81 @@
-<input [(ngModel)]="advertiserId"
-       placeholder="Advertiser ID" debugId="advertiser-id-input">
-<input [(ngModel)]="mediaPlanId"
-       placeholder="Media Plan ID" debugId="media-plan-id-input">
-<input [(ngModel)]="insertionOrderId"
-       placeholder="Insertion Order ID" debugId="io-id-input"><br>
-<div *ngFor="let choice of queryTypeChoices">
-    <input type="radio" [checked]="queryType===choice" (click)="queryType=choice" name="radio-group"
-           id="{{getQueryTypeShortName(choice)}}-radio-btn"/>
-    <label>{{getQueryTypeName(choice)}}</label><br>
+<div class="jumbotron d-flex align-items-center">
+    <div class="container-fluid">
+        <!-- accordion component for selecting query type and typing input parameter values-->
+        <bs-accordion [closeOthers]="true">
+            <bs-accordion-panel class='query-type-name'
+                                id="{{getQueryTypeShortName(choice)}}-panel"
+                                *ngFor="let choice of queryTypeChoices"
+                                heading="{{getQueryTypeName(choice)}}"
+                                [isOpen]="queryType===choice"
+                                (click)="queryType=choice">
+                <div class="section-name">{{requestSectionTitle}}</div>
+                <form class="form-group">
+                    <div>
```

Comment:
why doesnt this need an ngif like the other two

Reply:
Since advertiser id is required no matter what query type it is.